### ISSUE TITLE
[Bug]: Check my progress not working In prompt 1,4,5 [Generative AI with Vertex AI: Prompt Design]

### ISSUE THREAD

**Issue by Namasivaayam-L** [GoogleCloudPlatform/generative-ai#784]
### File Name

generative-ai/language/prompts/intro_prompt_design.ipynb

### What happened?

Check my Progress is not working even though I ran the entire notebook multiple times. Other prompts progress is detected and taken into account correctly. However, in these sections, the progress is not detected.
  1. Be concise
  2. Watch out for hallucinations
  3. Turn generative tasks into classification tasks to reduce output variability.     

I waited for the whole 1.5 hours as mentioned ("please wait") and still, the result was the same, I attempted the lab 2 times and I tried many variations by running sections and checking progress one by one.. but no hope at all... 

![image](https://github.com/GoogleCloudPlatform/generative-ai/assets/83755082/da768165-a5b3-4b60-a28c-3dddfb8c6a41)
![image](https://github.com/GoogleCloudPlatform/generative-ai/assets/83755082/63929cf2-96c6-43b4-85af-907c23a167f2)
![image](https://github.com/GoogleCloudPlatform/generative-ai/assets/83755082/f5557586-bc0e-44c7-af1b-1aad0014f5bb)
![image](https://github.com/GoogleCloudPlatform/generative-ai/assets/83755082/aca787c4-a736-4b1d-8efc-aaced1fb96ed)
![image](https://github.com/GoogleCloudPlatform/generative-ai/assets/83755082/869a9b84-c4c6-4dd6-af28-3efd27f64df9)

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by blamaestro** [comment]
I'm experiencing the same issue. Checks 1, 4 and 5 are failing although I ran the cells multiple times as well.

**Comment by yoary377** [comment]
I have the same issue

**Comment by Namasivaayam-L** [comment]
I have the same issue, with this lab as well: 
[https://www.cloudskillsboost.google/course_templates/981/labs/464008](https://www.cloudskillsboost.google/course_templates/981/labs/464008)

**Comment by polong-lin** [comment]
Hi folks, thank you for raising this. It seems like there's an issue with the activity tracker in these labs on Cloud Skills Boost, and it's not confirming that you've actually completed executing those cells.

I've filed a bug internally and will report back. 

Thanks for your patience and apologies for the trouble it's been causing.

**Comment by yoary377** [comment]
I've submitted a bug yesterday and tomorrow it was fixed. Please check one more time, for me the issue was fixed

**Comment by Namasivaayam-L** [comment]
thanks, guys.!!

**Comment by Namasivaayam-L** [comment]
> Hi folks, thank you for raising this. It seems like there's an issue with the activity tracker in these labs on Cloud Skills Boost, and it's not confirming that you've actually completed executing those cells.
> 
> I've filed a bug internally and will report back.
> 
> Thanks for your patience and apologies for the trouble it's been causing.

@polong-lin , Hey man.. thanks for filing and solving the bug. Can you please do the same for these issues as well, it would be a great help...!

1. https://github.com/GoogleCloudPlatform/generative-ai/issues/797
2. https://github.com/GoogleCloudPlatform/generative-ai/issues/796

**Comment by imsopov** [comment]
I'm having mine fail on all except the final 'Improve response quality by including examples'

![image](https://github.com/GoogleCloudPlatform/generative-ai/assets/515752/4b0109cc-a070-4ab1-a1d9-ad1ce7048e24)

**Comment by Asif-Sheriff** [comment]
Facing this problem again, except none of the "check progress"  buttons are working.
I migrated my notebook from "user managed notebooks" to "instances", has that got anything to do with this? 

and before you tell me to run the "recommended"  prompts, I have tried that multiple times, but does not seem to work.

Any help would be appreciated, thanks!
![image](https://github.com/GoogleCloudPlatform/generative-ai/assets/130328580/1a07dc7a-50cf-4cdb-9401-137e72c5adba)

**Comment by Namasivaayam-L** [comment]
> Facing this problem again, except none of the "check progress" buttons are working. I migrated my notebook from "user managed notebooks" to "instances", has that got anything to do with this?
> 
> and before you tell me to run the "recommended" prompts, I have tried that multiple times, but does not seem to work.
> 
> Any help would be appreciated, thanks! ![image](https://private-user-images.githubusercontent.com/130328580/346295423-1a07dc7a-50cf-4cdb-9401-137e72c5adba.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2MjY0NDMsIm5iZiI6MTcyMDYyNjE0MywicGF0aCI6Ii8xMzAzMjg1ODAvMzQ2Mjk1NDIzLTFhMDdkYzdhLTUwY2YtNGNkYi05NDAxLTEzN2U3MmM1YWRiYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxMFQxNTQyMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MmFiOTRjM2NhNTNlODIxMGFmMzNmOTg0ZTJkOWZiYWZkOTNlY2Q4MWY0MDZiNGU3MWI0YTJhZjA5MDQxYzY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.TgSrAncjJgYcnZBUONuIKB2DSAt3g8AHNheDi3iQW9I)

migrating yournotebook from "user managed notebooks" to "instances" may have resulted in this.. Any specific reason why you did this?

**Comment by danimajor1988** [comment]
Im having the same issue with this course, I can't get any tasks to mark as complete. I've run it over 3 times now for the full amount of time, 1:30, I've also watched several hours of tutorials about using all the features of Vertex AI because I thought it was me doing something wrong. Now that I know how the features work and can see others are having this same issue, I believe the original issue is not fixed like stated above.

**Comment by VarunJoshi** [comment]
I am having the same issue as well. Can't get my course to complete because of it :(

**Comment by suhaschandran** [comment]
I have the same problem with Try your own prompts section,

<img width="351" alt="Image" src="https://github.com/user-attachments/assets/8194a938-2a5a-4693-89f5-4a9b6ecfb98c" />

**Comment by nizhantiw** [comment]
<img width="668" alt="Image" src="https://github.com/user-attachments/assets/a903b2bf-fd0e-42a8-ab78-15f44955c2a8" />
Same issue

**Comment by enamacahiya** [comment]
Still receiving this issue in July 2025 with every "check my progress" button. Restarted computer, kernel and used incognito, guest profile, and non-incognito chrome. Also tried on both my mac and windows. Tried the backwards check that @vrnjoshi suggested. Nothing works. :(

**Comment by sheepzz1986** [comment]
Same issue since yesterday...I've tried a few rounds. Check my progress failed 

<img width="943" height="876" alt="Image" src="https://github.com/user-attachments/assets/c0fe8685-e412-4914-8590-66a90642aa63" />

**Comment by rtat23** [comment]
**Feb 2026** Same issue but with different lab: Recover VMs with Google Backup and DR Service. I've attempted 4 times, created new credentials with the name "Backup DR Appliance Service Account" and that still did not work. Have yet to hear a response from Coursera Customer Support for cert completion. 

Seems like the activity tracker is not reading the correct credential name, as I have looked at Youtube tutorials, and the credential name is different. It should be named "Backup DR Appliance Service Account" but is instead a copy of the Appliance name "qwiklabs-appliances-XXXX." Maybe this is the issue, idk? 

Any help would be greatly appreciated. Thank you!

<img width="1905" height="881" alt="Image" src="https://github.com/user-attachments/assets/2188b1fa-9ee1-444d-bbba-7f966184c855" />
